tests/test_boefjes_tasks.py

```diff
@@ -1,136 +1,126 @@
+        """
+    {
+        "count": 1,
+        "next": null,
+        "previous": null,
+        "results": [
+            {
+                "id": "1b20f85f-63d5-4baa-be9e-f3f19d6e3fae",
+                "hash": "19ed51514b37d42f79c5e95469956b05",
+                "scheduler_id": "boefje-_dev",
+                "p_item": {
+                    "id": "1b20f85f-63d5-4baa-be9e-f3f19d6e3fae",
+                    "hash": "19ed51514b37d42f79c5e95469956b05",
+                    "priority": 1,
+                    "data": {
+                        "id": "1b20f85f63d54baabe9ef3f19d6e3fae",
+                        "boefje": {
+                            "id": "dns-records",
+                            "name": "DnsRecords",
+                            "description": "Fetch the DNS record(s) of a hostname",
+                            "repository_id": null,
+                            "version": null,
+                            "scan_level": 1,
+                            "consumes": [
+                                "Hostname"
+                            ],
+                            "produces": [
+                                "DNSNSRecord",
+                                "DNSARecord",
+                                "DNSCNAMERecord",
+                                "DNSMXRecord",
+                                "DNSZone",
+                                "Hostname",
+                                "DNSAAAARecord",
+                                "IPAddressV4",
+                                "DNSSOARecord",
+                                "DNSTXTRecord",
+                                "IPAddressV6",
+                                "Network",
+                                "NXDOMAIN"
+                            ]
+                        },
+                        "input_ooi": "Hostname|internet|mispo.es.",
+                        "organization": "_dev"
+                    }
+                },
+                "status": "completed",
+                "created_at": "2022-08-09 11:53:41.378292",
+                "modified_at": "2022-08-09 11:54:21.002838"
+            }
+        ]
+    }
+    """
+    )
+
+    request = rf.get(reverse("task_list"))
+    request.user = user
+    request.active_organization = organization
+
+    response = BoefjesTaskListView.as_view()(request)
+
+    assertContains(response, "1b20f85f")
+    assertContains(response, "Hostname|internet|mispo.es.")
+
+    mock_scheduler_client.list_tasks.assert_has_calls([call(f"boefje-{organization.code}", limit=TASK_LIMIT)])
+
+
+def test_tasks_view_no_organization(rf, user):
+    request = rf.get(reverse("task_list"))
+    request.user = user
+    request.active_organization = None
+    request.session = "session"
+    request._messages = FallbackStorage(request)
+
+    response = BoefjesTaskListView.as_view()(request)
+
+    assertContains(response, "error")
+    assertContains(response, "Organization could not be found")
```

Comment:
For later/ some thoughts: I was just wondering if the response should also have a different status code